---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* BSc in Biochemistry, University of Coimbra, 2012-2015
* MSc in Cell and Molecular Biology (with a neurosciences specialisation), University of Coimbra, 2015-2017
  * Award for exceptional school performance (2017)
* PhD in Biological Sciences, European Bioinformatics Institute/University of Cambridge, 2017-2021 (ongoing)
  
Research experience
======
* Summer 2014: Research Intern 
  * *Chemistry Center of Coimbra/Biological Chemistry Group and Molecular Physical-Chemistry I&D Center* - **Universidade de Coimbra**
  * **Project A**: using NMR to characterise MRI agents (2 weeks)
    * *Supervisor: Maria Margarida Castro*
  * **Project B**: using vibrational spectroscopy to characterise polyamine complexes in cancer cell cultures
    * *Supervisor: Maria Paulo Matos Marques*

* Spring/Summer 2015: Research Intern (BSc. final year project)
  * *Laboratory of Immunology and Oncology* - **Universidade de Coimbra**
  * **Project**: studying the differential expression of PD-1/PD-L1 in invidividuals suffering from chronic myeloid leukaemia using flow cytommetry
    * *Supervisor: Paulo Rodrigues Santos*

* Winter 2016: Research Intern
  * *Biophysics Laboratory* - **Universidade de Coimbra**
  * **Project**: using a perfusion system to study how tolbutamide affects TEA-induced LTP, specifically regarding the release of zinc in the CA3 area of hippocampal slices
    * *Supervisor: Emilia Quintas-Ferreira*

* Spring 2016: Research intern
  * **Biocant**
  * Project: ODE models of the peroxiredoxin pathway in yeast using Wolfram Mathematica and COPASI
    * *Supervisor: Armindo Salvador*

* May 2016 - September 2017: Research intern (MSc. thesis)
  * **Center for Neuroscience and Cell Biology**
  * Project: using machine learning and computational tools to study the structure of protein complexes, with a particular focus on their interface. Developed tools that improve the prediction of hot-spots in protein interfaces and computational protocols for complexes with no known structure
    * *Supervisor: Irina S. Moreira*

* October 2017 - present: PhD student
  * **European Bioinformatics Institute**
  * Projects: 
    * Leveraging modern computer vision (with a particular focus on deep learning) to understand morphological differences in blood cells, with a focus on haematological cancers
        * Experience with high-throughput automatic slide scanners, particularly the Hammamatsu Nano Zoomer
    * Using longitudinal sequencing data and statistical modelling to capture the dynamics of somatic mutation evolution in individuals with clonal haematopoiesis
    * *Supervisor: Moritz Gerstung*
    * *Co-supervisor: George S. Vassiliou*

Publications
======
  <ul>{% for post in site.publications_previous %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  <ul>{% for post in site.publications_current %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>

Skills
======
* Programming 
  * *I have been programming actively for 4 years, with a quite large focus on machine learning, statistics and data science
  * Relevant languages
    * `Python` - advanced user
    * `R` - advanced user
    * `bash` - regular user
    * `C++` - beginner, currently learning
  * Deep learning (DL)
    * *A considerable part of my current work focuses on developing, implementing and using deep learning tools for classification/segmentation*
    * Relevant packages: `tensorflow` (`Python`), `pytorch` (`Python`)
  * Machine learning (ML)
    * Relevant packages: `scikit-learn` (`Python`), `caret` (`R`) 
  * Computer vision (CV)
    * Relevant packages: `opencv`, `scikit-image` (`Python`)

* Statistical analysis and data science
  * Application of both frequentist and Bayesian methods for inferential and predictive statistics
  * Development of approaches to disentangle the effects of different factors in specific problems, namely modelling longitudinal deep and targetted sequencing data
  * Supervised and unsupervised machine learning, with a particular focus on the former for the classification of hot-spots in protein-protein interfaces and haematologic disorders using whole blood slide imaging data
  * Development of a framework for multiple instance and vocabulary learning approach to create a "vocabulary of cells" for the classification of haematologic disorders

Language Skills
=====
  * Portuguese - proficient 
  * English - proficient
  * Spanish - beginner

Talks
======
  * **Oral presentations**
    * A Machine Learning Based Protein-Protein Hot-Spot Prediction Method - SpotOn. EJIBCE. Lisbon. (2016)
    * Using big-data to understand the protein interface landscape. EJIBCE. Coimbra. (2017)
    * Leveraging Automated Blood Cell Morphology for Myelodysplastic Syndrome Diagnosis and Prognosis Prediction. Quantitative BioImaging Conference. Oxford. (2020)
  * **Poster presentations**
    * The Natural History of Clonal Haematopoiesis. CRUK Cambridge Centre Early Detection Programme 6th Annual Symposium. Cambridge (2021)

Teaching
======
  <ul>{% for post in site.teaching %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
Service and leadership
======
* Co-founder of JEST, a junior enterprise focused on providing data science training
* Co-director of programming and radio broadcaster in Radio Universidade de Coimbra, a student-ran radio dedicated to teaching radio broadcasting to the general community
* Integrated several teams in student unions, with a particular interest in pedagogy and ecological and civil service

Other skills
=====
* Musical producer and member of a musical production collective, Ciga239, focused on artists with links to Coimbra
* Raspberry pi tinkering
